.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+env:
+  global:
+  - secure: 4d0AwdMSfwVEQprd1ZwJtTLhRBac54Dhw5G8CfWxt031XOn8upIMfI7PAyI3nGjWZ+6NKBdiyWGNmc7aew9UENdfH0OB5b1v2VfHYjaj4cDfPFS3dS9fP5JG4Q8FBkSwIj53Vhf/r+sfrTv2VgtPB5uLQmJxXyz2j4lm4TgzfPwH887egC3v6uFvjpigpIulh2jRLGYttt1AVNxT8Ep6ZdgQ+5ZaoFHkQ1Lk/OZG1zuuk9KGJsUzjRH+ICiA3c2c/CdnVqX+JEk6S5oigzymnV9BD4tVaxbRYqgp6kPWMtSRsFrqxJF65NfxYm+jVOX6g9eobQg9oEmhmmx8kRzEHrLUSCfbDFlKplN41AJnAFTejHPGqNUmDZA/MHPscwu6csl/37kPd/9XuvoPzSfmuetDL9iToSUNXJf1bV+LO/6zIZzJ3RkNo6tQvaTc729LM4l5eEC/E2Y7DOslve4OcZm0YuBx73VI+AaLBc+NYA0dJMGhXz/d0RS8P/46xJBhrcNrbKQ+XXr9iQCCe3nD680YUI7r+JxkytAQp12Za4s9T6bQRgfoL1KcDf6qJTchAXzTZY0S83nNCiTteMRQpSgybcNB2HGyfwR0GFwTpjYtbjgJf81HQHsc//gNzPXRiZ2Nh4uadJpXWpq9jr8Z/PUlndspyov4enaZMfvKpVY=
+  - secure: 2VwbiCHC7m9pEN3hSR0hpeqKwGi1ah4fqKhqr9K07aGzlw9trHQmCXQTSMiw8M0flT4AtpqWrxLWMnpHloin3UtkHEVoZv0LXvK7kCPrGFd8JXWDsHA9n3xIvVPR5jq6l2WCYF6OCDMdRWW8CYqsEeB6/aCJmjV9lQfe/JU5xy9R8LaHLYcTAwHH24q+8viqw4Kz04fRJ25lvKNHu6VUraa7XofhpTXOKhmqk+pPayI+1GICjHMYmJHWIeZJVT8E/0IK6fANewhJqY5pYZRwmnXni1p7u292ouK4aXhK3sqKaAIYs/HKvP13az/Kf8qljbvoESlpSyk04G7cLn5GT3w6Q1e+sgDpgzvjJloKPcSQyzAceUcA9xq7EbHPsboj+9ftEDeJDlA0sSRHKCjF+ZJXvVf8SK/uPCp2zevj/3A+bRPuOKVIh+MDosILkZaAu1mpqifragRgWM0w1FRJcCU2JPa1XodiusSHHgcKprtkJAvkCtdKH8emGEFM54drW+08WVJzyO1dPvwxZbvXeemyhjj5eUebJrELnkPsMT3GlwINqR1u59cF48Yp0hs/JMPWpOodKj0gtcVe7iMO852q5C5paYhaESrZxG/xV10WXJkMrwxVSAjk0DJ8FmwZeqgci6QQ84HxJOIB6PFvS80YfK7A9O90dENZXlnF6jo=
+python:
+- '2.6'
+- '2.7'
+install:
+- pip install -r requirements.txt
+script: nosetests
```

Comment:
I think `nosetests` is just example in Travis.
Will you install `nose` package? and is there any test for the script to run ?